src/widgets/principles/ui/principle-card/principle-card.scss

```
@@ -72,29 +25,43 @@
   display: flex;
   flex-flow: column nowrap;
   place-content: center flex-start;
-  align-items: center;
+  align-items: flex-start;
 
-  width: 380px;
-  height: 240px;
-  padding: 36px 24px 24px;
+  width: 100%;
+  padding: 23px;
 
   color: $color-gray-600;
 
   background-color: #fffaf0;
```

```suggestion
  background-color: $color-yellow-50;
```
I see these aren't your changes, but since you're fixing styles here, maybe we can use a color variable here?
`$color-yellow-50: hsl(40deg 100% 97%); // #fffaf0`

---

Fixed in https://github.com/rolling-scopes/site/pull/464/commits/0e90fe21ad3bff7ce81f6b84c5cd9261b28e23a5